docker/public/page.php

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta name="description" content="Some description meta." />
+    <meta charset="utf-8">
+    <title>Standard WOVN Test Page</title>
+</head>
+
+<body>
+
+<h1>Website Title (EN)</h1>
+
+<div class="container">
+    <p>This is a sample sentence.</p>
+</div>
+
+<ul>
+    <li>Item Number 1</li>
+    <li>このアイテムは意図的に日本語で書かれています</li>
```

Comment:
the source language will be english, so it's a bit strange to have japanese text on it

Reply:
The idea was to see if the backend can handle this relatively common scenario properly. But now I think about it it's probably not so much of the library's job and testing it here is a bit weird. What do you think, should we just remove it?

Reply:
the simpler the better, I think. We won't actually be doing any translation (if we do, we are mocking the html swapper response)

Reply:
If that's the case let me update all the page.html to be simpler.